agate-core/src/main/java/org/obiba/agate/repository/UserRepository.java

```diff
@@ -22,4 +22,6 @@
   List<User> findByRoleAndLastLoginLessThan(String role, DateTime dateTime);
 
   List<User> findByApplications(String application);
+
+  List<User> findByEmail(String email);
```

Comment:
We should force email to be unique, just like user name. Handle this when editing user details.